src/annotator/highlighter.ts

```diff
@@ -294,18 +295,49 @@ export function removeAllHighlights(root: HTMLElement) {
  * Remove highlights from a range previously highlighted with `highlightRange`.
  */
 export function removeHighlights(highlights: HighlightElement[]) {
+  setHighlightsFocused(highlights, false);
```

Comment:
I've elected to blanketly unfocus all highlights. We could more be more targeted and only unfocus SVG highlights if we wanted.